NEWS.md

```diff
@@ -6,6 +6,12 @@
 
 1. In `DT[, variable := value]`, when value is class `POSIXlt`, we automatically coerce it to class `POSIXct` instead, [#1724](https://github.com/Rdatatable/data.table/issues/1724). Thanks to @linzhp for the report, and Benjamin Schwendinger for the fix.
 
+## NEW FEATURES
+
+1. `fwrite()` with `compress="gzip"` produces compatible gz files when composed of multiple independent chunks owing to parallelization, [#6356](https://github.com/Rdatatable/data.table/issues/6356).  Earlier `fwrite()` versions could have issues with HTTP upload using `Content-Encoding: gzip` and `Transfer-Encoding: chunked`. Thanks to @oliverfoster for report and @philippechataignon for the fix.
```

Comment:
I wonder if this is new feature or bug fix. Not totally clear to me from the issue discussion -- it's not like decompression had any issues before. Leaving as a "feature" for now.

Reply:
I think the browser/web server HTTP gzip stuff is just a narrower implementation of the gzip compression algorithm than was utilised by fwrite. Such that some external utilities could decompress without issue and some could not. It would say fix, but that would depend on the initial intent of the compress flag, if it was only meant for internal use in R, reading and writing csv.gz files, the it is a feature to extend the implementation to support a wider ecosystem. I'm also happy to call it a feature (or fix).

Reply:
Thanks for the extra context. Yes, I think I'm most comfortable with "new feature" here -- it's extending `fwrite()` to "play better" with a wider set of tools for gzip files, where previously it could best be described as a more "pure" companion to `zlib`. Hopefully that makes sense :)